pkg/cmd/cli/describe/projectstatus.go

```diff
@@ -1170,6 +1181,30 @@ func (l *podLoader) AddToGraph(g osgraph.Graph) error {
 	return nil
 }
 
+type horizontalPodAutoscalerLoader struct {
```

Comment:
TODO to handle servers that don't have hpa.extensions, but do have hpa.autoscaling.